### DISCUSSION TITLE
Question about the issues with PUBG, Valorant, etc ..

### DISCUSSION THREAD

**Discussion by Adelscott** [mgth/LittleBigMouse#356] [Q&A]
I've seen that the issues  ( #255 #195 #207 ... ) affecting LMB and RMB in games are closed. Are they closed because the problem is resolved (IE : nothing to do, just install the latest version) or are they closed because we can now exclude programs in LBM ? That's unclear, the issues are closed without comment

**Comment by mgth** [comment]
Because we can exclude, and for now can't see another solution. But if some have an Idea, let's talk about.
